internal/sdkv2provider/provider_test.go

```diff
@@ -79,6 +79,17 @@ func testAccPreCheck(t *testing.T) {
 	}
 }
 
+func testAccPreCheckWithoutZoneID(t *testing.T) {
```

Comment:
default `testAccPreCheck` checks for Zone as well, but mTLS certs are account-level.